pkg/kf/manifest/manifest.go

```diff
@@ -54,7 +54,9 @@ func NewFromReader(reader io.Reader) (*Manifest, error) {
 
 	// TODO: validate manifest
 	m := Manifest{}
-	if err = yaml.Unmarshal(bytes, &m); err != nil {
+	if err = yaml.UnmarshalStrict(bytes, &m); err != nil {
+		fmt.Println(fmt.Errorf("Warning: manifest file contains unsupported config: %v", err))
```

Comment:
I think this should be an `Fprintf` to `stderr`

Reply:
done